site/frontend/src/pages/status_new/collector.vue

```diff
@@ -40,6 +41,37 @@ function formatJobStatus(status: BenchmarkJobStatus): string {
       return "Unknown";
   }
 }
+
+function ActiveStatus({collector}: {collector: CollectorConfig}) {
+  const now = new Date();
+  const maxInactivityHours = 1;
+  const lastHeartBeatAt = parseISO(collector.lastHeartbeatAt);
+  const hourDiff = differenceInHours(now, lastHeartBeatAt);
+  let statusText = "Active";
+  let statusClass = "active";
+
+  switch (collector.isActive) {
+    case true:
+      if (hourDiff >= maxInactivityHours) {
+        statusText = "Offline";
+        statusClass = "inactive";
+      } else {
+        statusText = "Active";
+        statusClass = "active";
+      }
+      break;
+    case false:
+      statusText = "Inactive";
+      statusClass = "inactive";
```

Comment:
Not that we should see this status often, but I'd use a grey color for "inactive", and a red color for "offline".

Reply:
<img width="507" height="302" alt="image" src="https://github.com/user-attachments/assets/821610d8-56d4-4577-9566-a541f4d24686" />